riptable/rt_str.py

```diff
@@ -815,6 +815,42 @@ def char(self, position: Union[int, List[int], np.ndarray]):
         out = out.view(f'{self._intype}1')
         return out
 
+    def strip(self, chars=' \t\r\n\v\f'):
+        # Convert the chars to strip to an array of uint8's.  This could probably be made cleaner.
+        to_strip = np.array([*chars.encode('ascii')], dtype='u1')
+        return self._apply_func(self.nb_strip, self.nb_strip_par, to_strip)
+
+    def _nb_strip(src, itemsize, dest, to_strip):

```

Comment:
suggestion: we could add two flags, `do_left, do_right` which trigger execution of the corresponding blocks 830-833, 835-839. It would then be trivial to add `lstrip` and `rstrip`